programs/clearing_house/src/math/margin.rs

```diff
@@ -291,10 +259,11 @@ pub fn calculate_margin_requirement_and_total_collateral(
     spot_market_map: &SpotMarketMap,
     oracle_map: &mut OracleMap,
     margin_buffer_ratio: Option<u128>,
-) -> ClearingHouseResult<(u128, i128, u128)> {
+) -> ClearingHouseResult<(u128, i128, u128, bool)> {
     let mut total_collateral: i128 = 0;
     let mut margin_requirement: u128 = 0;
     let mut margin_requirement_plus_buffer: u128 = 0;
+    let mut all_ = true;
```

Comment:
probs want a better name than all_ -- maybe oracle_is_ok ?

Reply:
lol that was a bad find/replace